.github/workflows/dependabot-commit-fix.yaml

```
@@ -0,0 +1,67 @@
+            WRAPPED=$(echo "$BODY" | fold -s -w 140)
+            if [ "$BODY" = "$WRAPPED" ]; then
+              echo "Body already compliant, nothing to fix."
+              exit 0
+            fi
+            NEW_BODY="$WRAPPED"
+          else
+            NEW_BODY=""
+            while IFS='|' read -r lib source target; do
+              short=$(basename "$lib")
+              NEW_BODY="${NEW_BODY}- ${short}: ${source} => ${target}
+"
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)